project/02-Modularity/Handout.md

```diff
@@ -0,0 +1,160 @@
+# Android Project - Part 2: Modularity
+
+In this second part of the SwEng project, you will add weather forecast capabilities to your app. Using an external service, [OpenWeatherMap](https://api.openweathermap.org/), you will display to the user a weather forecast at their location, or at an address of their choice. For that, you will also need to be able to find out where the user is located.
```

Comment:
this line says "or at an address of their choice" but the structure section below does not mention this